### DISCUSSION TITLE
无法使用node -v和npm-v,而且在vue2项目中无法使用npm命令。蹲一位大神

### DISCUSSION THREAD

**Discussion by fancyyyfff** [community/community#144179] [npm]
项目使用的是14.19.0版本的node，
nvm版本1.1.7
出于依赖问题和团队开发的考虑，暂时不考虑更改node版本

已验证:
环境变量由nvm设置，无误
nvm可安装的node版本也包括14.19.0
nvm可以install和use14.19.0
电脑的操作系统是window64位
14.19.0版本的node能够适应window64位

**Comment by obtar** [comment]
推荐通过官方的msi安装程序进行安装
![Screenshot_2025-05-02-11-35-51-88_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/user-attachments/assets/7edf532b-8367-4129-b678-1c1871470965)
